tests/ui/const-generics/legacy-const-generics-bad.stderr

```diff
@@ -1,10 +1,13 @@
 error[E0435]: attempt to use a non-constant value in a constant
   --> $DIR/legacy-const-generics-bad.rs:7:35
    |
-LL |     let a = 1;
-   |     ----- help: consider using `const` instead of `let`: `const a`
 LL |     legacy_const_generics::foo(0, a, 2);
    |                                   ^ non-constant value
+   |
+help: consider using `const` instead of `let`
+   |
+LL |     const a = 1;
```

Comment:
Wait, this suggestion is definitely wrong. Please mark it as *not* `MachineApplicable`.

Reply:
Sorry, I should've noticed it sooner. I'll look at your diffs closer next time, but maybe you should look at them closer too 😉 Group effort 🙌

Reply:
The sigh was at the suggestion being wrong in the first place :)